test/powershell/Modules/Microsoft.PowerShell.Utility/Clear-Variable.Tests.ps1

```diff
@@ -14,7 +14,7 @@ Describe "Clear-Variable DRT Unit Tests" -Tags "CI" {
 	It "Clear-Variable ReadOnly variable Name should throw exception and force Clear-Variable should works"{
 		Set-Variable foo bar -Option ReadOnly
 
-		$e = { Clear-Variable -Name foo -Scope 1 -EA Stop } | ShouldBeErrorId "VariableNotWritable,Microsoft.PowerShell.Commands.ClearVariableCommand"
+		$e = { Clear-Variable -Name foo -Scope 1 -EA Stop } | Should -Throw -ErrorId "VariableNotWritable,Microsoft.PowerShell.Commands.ClearVariableCommand"
```

Comment:
Please replace alias `-EA`.

I wonder - should we use `-PassThru`?

Reply:
-EA replaced. Yes, the -PassThru should be used. Now it is corrected.